README.rst

```diff
@@ -0,0 +1,34 @@
+
+
+==================
+Bioelectrics: Monodomain problem on a 2D domain with simple HodgkinHuxley1952 cell model
+==================
+
+Running the example
+===================
+Activate the openCMISS virtual environment for python bindings
+  source <path-to-opencmiss>/install/virtual_environments/oclibs_venv_py27_release/bin/activate
```

Comment:
Executable code should be placed in code blocks as per restructured text formatting.